### DISCUSSION TITLE
Uno AppBarButton not displaying label

### DISCUSSION THREAD

**Discussion by wkmhv** [unoplatform/uno#17520] [Q&A] [answered]
`<Grid>
  <Grid.RowDefinitions>
    <RowDefinition Height="Auto"></RowDefinition>
    <RowDefinition Height="*"></RowDefinition>
    <RowDefinition Height="35"></RowDefinition>
  </Grid.RowDefinitions>
  <StackPanel x:Name="HeaderPanel" Grid.Row="0" Orientation="Horizontal" Background="Aqua">
        <AppBarButton CornerRadius="2" Icon="OpenPane" Label="open" />
        <AppBarButton Icon="Like" Label="Like" Foreground="Green"/>
</StackPanel>
</Grid>
`
![grafik](https://github.com/user-attachments/assets/56963823-c768-4a4e-a290-77f11e26eddb)

**Comment by morning4coffe-dev** [comment]
Hey @wkmhv, which platform / what Uno version are you using? I pasted you code to a blank project and the labels are displayed on my side: 
![image](https://github.com/user-attachments/assets/e6051202-1e56-42ab-a8a4-c7e4819e9f5d)

**Comment by wkmhv** [comment]
![grafik](https://github.com/user-attachments/assets/c17fd6b9-d336-406d-8d3e-abddca4d4dbf)

![grafik](https://github.com/user-attachments/assets/e8812ce1-fad9-4770-879f-1fcd063e5da2)

**Reply by morning4coffe-dev** [reply]
Thank you for the clarification, unfortunately I am still not able to reproduce this behavior. The AppBarButtons in the last Row can get the label cut off as the height might be too small. Could you send me a full repro, so I can investigate further please?

**Reply by wkmhv** [reply]
[Uno5.zip](https://github.com/user-attachments/files/16236239/Uno5.zip)

**Reply by morning4coffe-dev** [reply, marked as answer]
I see! Thank you for the full repro. The issue is didn't replicate on my side as I didn't know you are using Material theme for your app. 

You have a few options:
1. You can use some other Material control, e.g. https://platform.uno/docs/articles/external/uno.toolkit.ui/doc/controls/TabBarAndTabBarItem.html
2. If you don't want Material for your app, you can remove the `um:MaterialTheme` from your `App.xaml`
3. Or you can use a combination of both styles, I have just recently added a helper to our Gallery - https://github.com/morning4coffe-dev/Uno.Gallery/blob/8a5b709b7f1ee5483e39859aa1f9cc4e5def985e/Uno.Gallery/Uno.Gallery.Shared/Helpers/FluentHelper.cs - you can add the FluentHelper.cs to your project and ensure the desired control is using Fluent styles

**Reply by wkmhv** [reply]
Thank's I will try
